docs/tools/relayer-developer.md

```diff
@@ -28,18 +28,132 @@ Please fill out [this form](https://forms.gle/rhWA25m3jjuPNPva9) to request acce
+    anotherProperty: {
+        sdfsdf: 123456
+    }
+})
+
+const verfiableUriIdentifier = '0x0000'
+
+// Get the bytes4 representation of the verification method
+const verificationMethod = web3.utils.keccak256('keccak256(utf8)').substr(0, 10)
+> '0x6f357c6a'
+
+// Get the hash of the JSON file (verification data)
+const verificationData = web3.utils.keccak256(json)
+> '0x820464ddfac1bec070cc14a8daf04129871d458f2ca94368aae8391311af6361'
+
+// Get the verification data length and padd it as 2 bytes
+const verificationDataLength = web3.utils.padLeft(web3.utils.numberToHex((verificationData.substring(2).length) / 2), 4);
+> 0x0020
+
+// store the JSON anywhere and encode the URL
+const url = web3.utils.utf8ToHex('ifps://QmYr1VJLwerg6pEoscdhVGugo39pa6rycEZLjtRPDfW84UAx')
+> '0x696670733a2f2f516d597231564a4c776572673670456f73636468564775676f3339706136727963455a4c6a7452504466573834554178'
+
+
+// final result (to be stored on chain)
+const VerfiableURI =  verfiableUriIdentifier + verificationMethod.substring(2) + verificationDatalength.substring(2) + verificationData.substring(2) + url.substring(2)
+                      ^                        ^                                 ^                                     ^                               ^
+                      0000                     6f357c6a                          0020                                  820464ddfac1be...               696670733a2f2...
+
+// structure of the VerifiableURI
+0x0000 + 6f357c6a +       0020 +                    820464ddfac1bec070cc14a8daf04129871d458f2ca94368aae8391311af6361 + 696670733a2f2f516d597231564a4c776572673670456f73636468564775676f3339706136727963455a4c6a7452504466573834554178
+  ^      ^                ^                         ^                                                                  ^
+  0000   keccak256(utf8)  verificationDatalength    verificationData                                                   encoded URL
+
+// example value
+0x00006f357c6a0020820464ddfac1bec070cc14a8daf04129871d458f2ca94368aae8391311af6361696670733a2f2f516d597231564a4c776572673670456f73636468564775676f3339706136727963455a4c6a7452504466573834554178
+```
+
+ℹ️ More info on [VerifiableURI](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md#verifiableuri)
+
+</details>
+
+<details>
+  <summary>OPTION 2: salt & postDeploymentCallData</summary>
+
+**Why is it useful to deploy UP to pass salt and postDeploymentCallData?**
+
+To be able to deploy a UP on the same address across different chains.
+
+**How to generate each parameter?**
+
+```javascript title="salt: A 32 bytes salt used to compute the deployment address of the contract."
+const salt = '0x' + crypto.randomBytes(32).toString('hex');
+```
+
+```javascript title="postDeploymentCallData: Call data which will be executed on the ERC725Account contract after deployment. Should contain the encoded setData transaction to set required permission setting and LSP3 Profile Data"
```

Comment:
Is it not setDataBatch?

```suggestion
```javascript title="postDeploymentCallData: Calldata which will be executed on the ERC725Account contract after deployment. Should contain the encoded setDataBatch transaction to set the initial permissions and LSP3 Profile Data"
```